Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasketViewController.m

```diff
@@ -31,15 +35,51 @@ - (IBAction)nextButtonTapped:(id)sender {
     //switch tab controller to index 0
 }
 
-/*
-#pragma mark - Navigation
+- (IBAction)headerImageButtonTapped:(id)sender {
+        UIImagePickerController *imagePickerVC = [UIImagePickerController new];
+        imagePickerVC.delegate = self;
+        imagePickerVC.allowsEditing = YES;
+        BOOL cameraSourceAvail = [UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera];
+        BOOL photoLibAvail = [UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypePhotoLibrary];
+        if (cameraSourceAvail && photoLibAvail) {
+            //UIActionSheet to pick a source type
+            UIAlertController *sourceTypePicker = [UIAlertController alertControllerWithTitle:@"" message:@"" preferredStyle:UIAlertControllerStyleActionSheet];
+            UIAlertAction *camera = [UIAlertAction actionWithTitle:@"Camera" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+                imagePickerVC.sourceType = UIImagePickerControllerSourceTypeCamera;
+            }];
+            UIAlertAction *photoLib = [UIAlertAction actionWithTitle:@"Photo Library" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+                imagePickerVC.sourceType = UIImagePickerControllerSourceTypePhotoLibrary;
+            }];
+            [sourceTypePicker addAction:camera];
+            [sourceTypePicker addAction:photoLib];
+            [self presentViewController:sourceTypePicker animated:YES completion:nil];
+        } else {
+            imagePickerVC.sourceType = UIImagePickerControllerSourceTypePhotoLibrary;
+        }
+        [self presentViewController:imagePickerVC animated:YES completion:nil];
+    }
+
+- (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary<NSString *,id> *)info {
+    UIImage *originalImage = info[UIImagePickerControllerOriginalImage];
```

Comment:
Same feedback as before, you should avoid unused variables like `originalImage`.